### PR TITLE
Add a few high-impact CodeWorld projects that were missing from the first draft

### PR DESCRIPTION
**Pull request by cdsmith**

### CODE REVIEW

**Review by gwils — APPROVED** [review]

File: content/ideas/codeworld.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ title: Improve the CodeWorld environment for Haskell in K-12 education
...
+
+* **Finish collaborative editing.** Diffculty: Intermediate.  There is [code already written](https://github.com/google/codeworld/pull/551) to add collaborative editing (in the style of Google Docs) to CodeWorld, and [CodeWorld Issue #17](https://github.com/google/codeworld/issues/17) has discussion and advice.  This could be a game-changer for groupwork in early programming classes.  The existing pull request is (or at least was at one time) a working draft of the feature, but it's not mergable in its current form, and someone willing to drive this project to completion could have a lot of impact.  This would be a good project for a student who is either comfortable with or motivated to learn web technologies like JavaScript, WebRTC, the CodeMirror editor, etc., but also wants to work on interesting algorithms such as operational transforms.
```
Comment (emilypi) [inline content/ideas/codeworld.md]: Typo: `Diffculty` -> `Difficulty`

Comment (emilypi) [inline content/ideas/codeworld.md]: Typo: `mergable` -> `mergeable`

**Review by emilypi — COMMENTED** [review]
Looks good - just a few typo's to fix then we're all set!

**Comment by cdsmith** [comment]
Typos fixed.  Thanks, @emilypi

**Review by emilypi — APPROVED** [review]
Thanks @cdsmith!

**Pull request merged**